liubov.tarasenko/homework6/link.rb

```diff
@@ -0,0 +1,10 @@
+require_relative 'document'
+require 'nokogiri'
```

Comment:
You don't need to require this as is already imported from 'document.rb'

Reply:
Thank you. Have confused with js.